plenum/server/consensus/ordering_service.py

```diff
@@ -965,44 +957,29 @@ def _request_missing_three_phase_messages(self, view_no: int, seq_frm: int, seq_
             self._request_commit(key)
 
     def _request_three_phase_msg(self, three_pc_key: Tuple[int, int],
-                                 stash: Dict[Tuple[int, int], Optional[Tuple[str, str, str]]],
                                  msg_type: str,
                                  recipients: Optional[List[str]] = None,
-                                 stash_data: Optional[Tuple[str, str, str]] = None) -> bool:
-        if three_pc_key in stash:
-            self._logger.debug('{} not requesting {} since already '
-                               'requested for {}'.format(self, msg_type, three_pc_key))
-            return False
-
-        # TODO: Using a timer to retry would be a better thing to do
-        self._logger.trace('{} requesting {} for {} from {}'.format(
-            self, msg_type, three_pc_key, recipients))
-        # An optimisation can be to request PRE-PREPARE from f+1 or
-        # f+x (f+x<2f) nodes only rather than 2f since only 1 correct
-        # PRE-PREPARE is needed.
-        self._request_msg(msg_type, {f.INST_ID.nm: self._data.inst_id,
-                                     f.VIEW_NO.nm: three_pc_key[0],
-                                     f.PP_SEQ_NO.nm: three_pc_key[1]},
-                          frm=recipients)
-
-        stash[three_pc_key] = stash_data
-        return True
+                                 stash_data: Optional[Tuple[str, str, str]] = None):
```

Comment:
I think `stash_data` is not self-descriptive here. It looks like this is rather the parameters we would like to have in the Reply.